# HG changeset patch
# User jwe
# Date 886493239 0
#      Tue Feb 03 08:07:19 1998 +0000
# Node ID d0384ceed762915cf2154b824a421ee47b95bb5b
# Parent  1e208b4d846de842d61a10b7faa96ae42c81e65c
[project @ 1998-02-03 08:07:19 by jwe]
matrix.exp

diff --git a/test/octave.test/matrix/matrix.exp b/test/octave.test/matrix/matrix.exp
--- a/test/octave.test/matrix/matrix.exp
+++ b/test/octave.test/matrix/matrix.exp
@@ -261,31 +261,31 @@ do_test logspace-2.m
 set test logspace-3
 set prog_output "usage:.*"
 do_test logspace-3.m
 
 set test logspace-4
 set prog_output "usage:.*"
 do_test logspace-4.m
 
-set test hadamard-1
+set test sylvester_matrix-1
 set prog_output "ans = 1"
-do_test hadamard-1.m
+do_test sylvester_matrix-1.m
 
-set test hadamard-2
+set test sylvester_matrix-2
 set prog_output "error:.*"
-do_test hadamard-2.m
+do_test sylvester_matrix-2.m
 
-set test hadamard-3
+set test sylvester_matrix-3
 set prog_output "usage:.*"
-do_test hadamard-3.m
+do_test sylvester_matrix-3.m
 
-set test hadamard-4
+set test sylvester_matrix-4
 set prog_output "usage:.*"
-do_test hadamard-4.m
+do_test sylvester_matrix-4.m
 
 set test hankel-1
 set prog_output "ans = 1"
 do_test hankel-1.m
 
 set test hankel-2
 set prog_output "error:.*"
 do_test hankel-2.m
